modular_zubbers/code/modules/borgs/code/robot_model.dm

```diff
@@ -25,9 +25,31 @@
 	if (model_features && (TRAIT_R_LIGHT_WEIGHT in model_features))
 		cyborg.can_be_held = TRUE
 		cyborg.held_w_class = WEIGHT_CLASS_HUGE
+		cyborg.add_traits(list(TRAIT_CATLIKE_GRACE), INNATE_TRAIT)
+		cyborg.mob_size = MOB_SIZE_SMALL
 	else
 		cyborg.can_be_held = FALSE
 		cyborg.held_w_class = WEIGHT_CLASS_NORMAL
+		cyborg.remove_traits(list(TRAIT_CATLIKE_GRACE), INNATE_TRAIT)
+		cyborg.mob_size = MOB_SIZE_HUMAN
+
+// To load the correct walking sounds with out removing them
+/obj/item/robot_model/proc/update_footsteps()
+	var/mob/living/silicon/robot/cyborg = robot || loc
+	if (!istype(robot))
+		return
+	if (model_features && ((TRAIT_R_SQUADRUPED in model_features) || (TRAIT_R_TALL in model_features) || (TRAIT_R_WIDE in model_features)))
+		if (model_features && (TRAIT_R_SQUADRUPED in model_features))
+			cyborg.AddElement(/datum/element/footstep, FOOTSTEP_ROBOT_SMALL, 6, -6, sound_vary = TRUE)
+		if (model_features && (TRAIT_R_TALL in model_features))
+			cyborg.AddElement(/datum/element/footstep, FOOTSTEP_MOB_SHOE, 2, -6, sound_vary = TRUE)
```

Comment:
These should probably be else ifs just so we don't try to add footsteps twice

Reply:
I tried it and it breaks when doing such and applies both.